lib/IRremoteESP8266/IRremoteESP8266.cpp

```diff
@@ -517,22 +570,22 @@ void ICACHE_FLASH_ATTR IRsend::sendRC6(uint64_t data, uint16_t nbits,
     // Header
     mark(RC6_HDR_MARK);
     space(RC6_HDR_SPACE);
-    mark(RC6_T1);  // Start bit
+    // Start bit.
+    mark(RC6_T1);  // mark, then space == 0x1.
     space(RC6_T1);
-    uint16_t t;
     // Data
-    for (uint64_t i = 0, mask = 1ULL << (nbits - 1); mask; i++, mask >>= 1) {
-      // The fourth bit we send is a "double width trailer bit".
-      if (i == 3)
-        t = 2 * RC6_T1;  // double-wide trailer bit
+    uint16_t bitTime;
+    for (uint64_t i = 1, mask = 1ULL << (nbits - 1); mask; i++, mask >>= 1) {
```

Comment:
Ah. Yes. I did do that deliberately. Well spotted. _i_ is used only for working out which is the _trailer_ bit. _mask_ is  the true iterator/limiter. I normally would have started at 0, but as you see on the line below, the comment indicates that it operates on the _fourth bit_. Previously when it started at 0, to do that it required using _(i == 3)_, that didn't gel well with the comment. So to avoid people thinking it might be a bug, I decided in that loop to start at _1_ to better fit the comment and avoid confusion. Seems that no good deed goes unpunished. ;-)

@rahuljawale Thanks for again being cautions and observant.